src/js/player.js

```diff
@@ -516,14 +532,14 @@ class Player extends Component {
       height = this.height_;
     } else {
       // Otherwise calculate the height from the ratio and the width
-      height = width  * ratioMultiplier;
+      height = width * ratioMultiplier;
     }
 
     // Ensure the CSS class is valid by starting with an alpha character
-    if (/^[^a-zA-Z]/.test(this.id())) {
-      idClass = 'dimensions-'+this.id();
+    if ((/^[^a-zA-Z]/).test(this.id())) {
+      idClass = `dimensions-${this.id()}`;
```

Comment:
It complained about the infix operator not having spaces; so, I figured I'd just switch to the template string. Can switch back if desired.